install/local_setup.sh

```
@@ -0,0 +1,137 @@
+# generated from colcon_core/shell/template/prefix.sh.em
```

Shellcheck (0.10.0)

[error] Tips depend on target shell and yours is unknown. Add a shebang or a 'shell' directive.

(SC2148)